### PR TITLE
Edited get started

### PR DESCRIPTION
**Pull request by RichardLitt**
@nicola How does this look? Happy to keep going with this type of writing if it looks good!

### CODE REVIEW

File: Get-Started.md [hunk 1]
```diff
@@ -1,44 +1,45 @@
...
-- This tutorial will guide you from the very basics to real example of IPLD.
-- IPLD stands for the Inter Planetary Linked Data
-- When we get data from the internet, we know it comes from the right place, if the URL is https (some times not even! TODO: add symantec & fake certificates), but we don’t know if it was the data we intended to get!
-- The same problem is with links, we don’t know if a link is still linking to the right page
-- Finally, the general problem of the current web is that we trust the source to give us the right data, so other computers in the network cannot really help sharing the same data!
-- These 3 properties are secure linking, distributed naming and immutable content!
-- Where does IPLD fit into this? IPLD is a data format to describe datasets where data can link to other data via cryptographic hashing, IPLD also provides a path scheme to point to some data in the datasets.
-- Let’s unroll what we just said.
+
+This tutorial will teach you about how to use IPLD. We'll start from the very basics, and end with some examples.
```
Comment (nicola) [inline Get-Started.md]: I would prefer not to give the idea of "we teach you how to use our tools", but "you learn how to" or "we guide through". Does that make sense?

File: Get-Started.md [hunk 2]
```diff
@@ -1,44 +1,45 @@
...
+
+IPLD stands for InterPlanetary Linked Data. This is based off of [IPFS](https://ipfs.io), which is the InterPlanetary File System. IPFS starts with a desire to have filesystems that work even when there are long delays involved (such as would be the case with interplanetary space travel). IPLD builds on top of this by allowing us to reference data in IPFS - or elsewhere.
```
Comment (nicola) [inline Get-Started.md]: "interplanetary space travel!" :)
It is too exciting not to put a !
(your choice!)

File: Get-Started.md [hunk 3]
```diff
@@ -1,44 +1,45 @@
...
+
+When we get data from the internet by accessing a webpage in our browser, we know it comes from the right place if the URL is https. This is because the HTTPS protocol gives some security that the data hasn't been compromised, for instance by a man-in-the-middle attack. (Of course, sometimes not even HTTPS is enough! TODO: add symantec & fake certificates). However, while we know that the data comes from the right place, we _don’t_ know if it was the data we intended to get!
```
Comment (nicola) [inline Get-Started.md:7]: " that the data hasn't been compromised, for instance by a man-in-the-middle attack" not sure

HTTPS gives guarantees that the data comes from the right place (the owner of the domain)

File: Get-Started.md [hunk 4]
```diff
@@ -1,44 +1,45 @@
...
-- If we ask to a network (like the ipfs network) for a hash
-- then, they cannot lie about the content that they are giving
-- since you can generate the hash of the content received and check if it is the same one to the one that you asked for!
+
+In IPLD, we refer to objects with their cryptographic hash. A cryptographic hash is the result of a [one way function](https://en.wikipedia.org/wiki/One-way_function). For any given input, it returns a string of bits. Thst string is always the same size as any other returned string by the function, and looks like this: `QmT78zSuBmuS4z925WZfrqQ1qHaJ56DQaTfyMUF7F8ff5o`. Normally, the string is a combination of symbols that look random. Importantly, every object has only one cryptographic hash that can be returned by a given cryptographic function. This means you could put the same input through a cryptographic function a million times, and it would always return the same result.
+
+Also importantly, it is impossible (well, really really difficult) to tell, given a hash, what content it refers to. That is why it is a one way function - data doesn't flow both ways. You also can't tell easily, given two hashes, if the content was similar.
```
Comment (nicola) [inline Get-Started.md]: "tell easily",
drop easily, you can't tell at all!

**Review by nicola — CHANGES_REQUESTED** [review]

File: Get-Started.md (hunk 3 above)
Comment (RichardLitt) [reply Get-Started.md:7]: Which is what a man in the middle attack would compromise. See opening paragraphs, here: https://en.wikipedia.org/wiki/HTTPS.

**Comment by RichardLitt** [comment]
@nicola Added more edits.

**Comment by daviddias** [comment]
New website is now on this PR https://github.com/ipld/website/pull/30, closing this one.

**Pull request closed without merging**